eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -20,6 +20,9 @@ parameters:
   - name: CoverageReportPath
     type: string
     default: 'sdk/*/*/*cov_xml.xml'
+  - name: FailOnSpellingErrors
+    type: boolean
+    default: false
```

Comment:
Is there a reason to add this until we ready to set it to true in at lease one place? 

Also we seem to be adding in some double negatives (i.e. `not(Fail..)`) I wonder if we if we should call this `ContinueOnSpellingErrors` just to align with what we are passing it to.

Reply:
I had other names which were worse... `ContinueOnSpellingErrors` is much better than anything I had this morning.

Reply:
Also, the reason to add this is to give people the ability to opt into it when ready.